person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @id = id
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    if parent_permission || of_age?
+      puts true
+    else
+      puts false
+    end
+  end
+
+  private
+
+  def of_age?
+    if @age >= 18
+      puts true
+    else
+      puts false
+    end
+  end
```

Comment:
- [ ] For `can_use_services?` and `of_age?` methods you suppose to only return `true` or `false` values not printing it too. So please fix that.
- [ ] Please add a `@` before `parent_permission` variable in `can_use_services?` method because it's an instance variable and when you try to use this method you will face an error. so please fix that

[Hint]: you can do it just like that:
```ruby
def can_use_services?
    @parent_permission || of_age?
  end

  private

  def of_age?
    @age >= 18
  end
```